src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -293,13 +293,17 @@ static int path_is_busy(const char *where) {
 
         /* not a mountpoint but it contains files */
         r = dir_is_empty(where, /* ignore_hidden_or_backup= */ false);
-        if (r < 0)
+        if (r == -ENOTDIR) {
+                log_debug("\"%s\" is not a directory, ignoring.", where);
+                return true;
+        } else if (r < 0)
```

Comment:
Unneeded `else`

Reply:
It's there on purpose to avoid extra lines. The whole thing is more readable with each indented block preceded by exactly one line with a conditional.